modules/data-entry/src/main/frontend/src/dataHomepage/Forms.jsx

```diff
@@ -102,41 +72,23 @@ function Forms(props) {
       "format": "string",
     },
   ]
-  const actions = [
-    DeleteButton
-  ]
 
   return (
-    <Card>
-      <CardHeader
-        color={"warning"/* Does nothing */}
-        title={
-          <Button className={classes.cardHeaderButton}>
-            {title}
-          </Button>
-        }
-        action={
-        <div className={classes.mainPageAction}>
-          <NewFormDialog presetPath={questionnairePath}>
-            New form
-          </NewFormDialog>
-        </div>
-        }
-        classes={{
-          action: classes.newFormButtonHeader
-        }}
-      />
-      <CardContent>
-        <LiveTable
+    <Grid container direction="column" spacing={4}>
+      <Grid item className={classes.dashboardEntry}>
+        <Typography variant="h2">Forms</Typography>
+      </Grid>
+      <Grid item>
+        <FormView
+          expanded
+          disableAvatar
+          topPagination
+          title=""
           columns={columns}
-          customUrl={customUrl}
-          filters
-          joinChildren="lfs:Answer"
-          actions={actions}
-          entryType="Form"
-          />
-      </CardContent>
-    </Card>
+          questionnaire={questionnaireID}
```

Comment:
Source of the individual-questionnaire-view bug
```suggestion
          questionnaire={questionnaireID[1]}
```